shim/src/sandbox.js

```diff
@@ -8,17 +12,24 @@ function createIframe() {
     return el;
 }
 
-export function createSandbox(customGlobalObj, customThisValue) {
+export function createSandbox() {
```

Comment:
the `createSandbox` method was split to create the sandbox and eventually set the global and thisValue.